data/data-pipeline/data_pipeline/etl/score/etl_score_geo.py

```diff
@@ -1,6 +1,7 @@
 import math
 import pandas as pd
 import geopandas as gpd
+import numpy as np
```

Comment:
A few notes on this file in places I haven't changed:

1. In line 73, there's a `use_cols` param to the gpd read file call, but I don't think this argument can be used. Can I delete this? 
2. In line 86, a geoid column is renamed in place. I'm wondering if there's a reason? I'm also wondering why the merge can't take place on indexes (like casting the GEOID Tract as an index and then merging with `left_index=True, right_index=True`. I'm just curious!
3. In line 148 (or thereabouts), the rounding procedure takes a series. I'm wondering why it's not just an integer?

Reply:
Would love for @esfoobar-usds to weigh in as well!

1. Seems like you're right so you can probably drop it. 

2. Actually as part of `pandas-vet` compliance we'll be moving away from all uses of `inplace=True` (I was guilty of a lot of these, though not specifically this one, before realizing it was frowned upon). So you can make that change. 

2b. Merging on indices would improve speed and it sounds fine to me as long as you reset the index (or otherwise write the output with index=True) to make sure downstream the data stays the same.

3. I'm not super familiar with this code but I *think* what is happening is that it's only rounding the `self.TARGET_SCORE_RENAME_TO` column. Which could also perhaps happen in a more direct / obvious fashion.

Reply:
1 and 2, 2b sound good. For 3, I meant something along the lines of df[column].round(2), which should accomplish the same thing (I believe!) as df[column].round(pd.Series([2,2,2,2,2,2...]) and would be easier to read?

Reply:
Oh I see. If that's how that works (I haven't checked documentation), that sounds more readable to me so I'd endorse it!

Reply:
I'm good with all your suggestions @emma-nechamkin!